packages/EasyErrorHandler/src/Bridge/Symfony/Interfaces/ErrorCodesProviderInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Interfaces;
```

Comment:
I believe we should place it under `EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes`. Wdyt?

Reply:
Agree, Done.